web/src/containers/Investments.js

```diff
@@ -0,0 +1,95 @@
+import React, { useEffect, useState } from "react";
+import { Grid, Icon, makeStyles, Typography } from "@material-ui/core";
+import SimpleTable from "../components/SimpleTable";
+import MonetizationOnTwoToneIcon from "@material-ui/icons/MonetizationOnTwoTone";
+import { yellow } from '@material-ui/core/colors';
+import InvestmentCard from "../components/InvestmentCard";
+import InvestmentsService from "../services/investments.service";
+
+const columns = [
+  { field: "loan", headerName: "Loan", width: 160, headerClassName: 'data-grid-header'},
+  { 
+    field: "tokens",
+    headerName: "Tokens",
+    type: "number",
+    width: 120,
+    headerClassName: 'data-grid-header',
+    // eslint-disable-next-line react/display-name
+    renderCell: (params) => (
+      <div style={{display: 'flex', alignItems:'center'}}>
+        {params.value}
+        <Icon style={{marginLeft: 4}}>
+          <MonetizationOnTwoToneIcon style={{color: yellow[700]}}/>
+        </Icon>
+      </div>
+    )
+  },
+  { field: "phase", headerName: "Phase", headerClassName: 'data-grid-header'},
+  { field: "inMarketplace", headerName: "Listed in the Market", type: "number",  width: 180, headerClassName: 'data-grid-header'}
+]
+
+const Investments = () => {
+  const [investments, setInvestments] = useState([])
+  const [selected, setSelected] = useState({loan: 'Mock'})
+
+  useEffect(() => {
+    async function getResponseFromAPI() {
+      return (await InvestmentsService.getPersonal())
+    }
+    
+    getResponseFromAPI()
+      .then((response) => {
+        setInvestments(response)
+        setSelected(response[0])
+      })
```

Comment:
move to getResponseFromAPI() ?

Reply:
Doesn't work, .then() triggers when a returned promise from an async function is resolved.

Reply:
Example:
async function fetchRequests() {
      const data = await LoanRequestService.getPendingLoanRequests();
      setRequests(data);
      setLoading(false);
    }
fetchRequests();